pkgs/by-name/lr/lrcget/package.nix

```diff
@@ -76,30 +80,25 @@ rustPlatform.buildRustPackage rec {
       alsa-lib
     ];
 
+  # To fix `npm ERR! Your cache folder contains root-owned files`
+  makeCacheWritable = true;
+
   # Disable checkPhase, since the project doesn't contain tests
   doCheck = false;
 
-  postInstall = ''
-    install -DT icons/128x128@2x.png $out/share/icons/hicolor/128x128@2/apps/lrcget.png
-    install -DT icons/128x128.png $out/share/icons/hicolor/128x128/apps/lrcget.png
-    install -DT icons/32x32.png $out/share/icons/hicolor/32x32/apps/lrcget.png
-  '';
-
-  # WEBKIT_DISABLE_COMPOSITING_MODE essential in NVIDIA + compositor https://github.com/NixOS/nixpkgs/issues/212064#issuecomment-1400202079
-  postFixup = ''
-    wrapProgram "$out/bin/lrcget" \
-      --set WEBKIT_DISABLE_COMPOSITING_MODE 1
-  '';
-
-  desktopItems = [
-    (makeDesktopItem {
-      name = "LRCGET";
-      exec = "lrcget";
-      icon = "lrcget";
-      desktopName = "LRCGET";
-      comment = meta.description;
-    })
-  ];
+  # don't automatically wrap binary to avoid double wrapping it
+  dontWrapGApps = true;
+
+  postFixup =
+    # WEBKIT_DISABLE_COMPOSITING_MODE essential in NVIDIA + compositor https://github.com/NixOS/nixpkgs/issues/212064#issuecomment-1400202079
+    lib.optionalString (!stdenv.hostPlatform.isDarwin) ''
+      wrapGApp "$out/bin/LRCGET" \
+        --set WEBKIT_DISABLE_COMPOSITING_MODE 1
```

Comment:
Why not

```nix
    gappsWrapperArgs+=(
      # WEBKIT_DISABLE_COMPOSITING_MODE essential in NVIDIA + compositor https://github.com/NixOS/nixpkgs/issues/212064#issuecomment-1400202079
      --set WEBKIT_DISABLE_COMPOSITING_MODE 1
    )
```

instead?

Reply:
True, I'll change it once I get back home